tasks/pack.js

```diff
@@ -7,12 +7,18 @@ import args from './lib/args';
 gulp.task('pack', ['build'], () => {
   let name = packageDetails.name;
   let version = packageDetails.version;
-  let filename = `${name}-${version}.zip`;
-  return gulp.src('dist/**/*')
+  let ext = {
+    chrome: 'zip',
+    opera: 'zip',
+    moz: 'xpi',
```

Comment:
Firefox requires a filename of packaged extension ends with `.xpi`, not `.zip`. (https://wiki.mozilla.org/WebExtensions#Packaging)